README.md

```diff
@@ -16,180 +18,72 @@ Streisand
-
-### Running Streisand on Other Providers (Advanced) ###
-
-You can also run Streisand on a new Ubuntu 16.04 server. Dedicated hardware? Great! Esoteric cloud provider? Awesome! To do so, simply choose "Existing Server (Advanced)" from the menu after running `./streisand` and provide the IP address of the existing server when prompted.
+* [OpenSSH](https://www.openssh.com/)
+    * [Tinyproxy](https://banu.com/tinyproxy/) may be used as an HTTP proxy.
+* [OpenConnect](https://ocserv.gitlab.io/www/index.html) / [Cisco AnyConnect](https://www.cisco.com/c/en/us/products/security/anyconnect-secure-mobility-client/index.html)
+    * This protocol is widely used by multi-national corporations, and might not be blocked.
+* [OpenVPN](https://openvpn.net/index.php/open-source.html)
+    * [Stunnel](https://www.stunnel.org/index.html) add-on available.
+* [Shadowsocks](https://shadowsocks.org/en/index.html), 
+    * [simple-obfs](https://github.com/shadowsocks/simple-obfs) add-on available.
+* A private [Tor](https://www.torproject.org/) bridge relay
+    * [Obfsproxy](https://www.torproject.org/projects/obfsproxy.html.en) with obfs4 available as an add-on.
+* [WireGuard](https://www.wireguard.com/), a modern high-performance protocol.
 
-The server must be accessible using the `$HOME/.ssh/id_rsa` SSH Key, and **root** is used as the connecting user by default. If your provider requires you to SSH with a different user than root (e.g. `ubuntu`) specify the `ANSIBLE_SSH_USER` environmental variable (e.g. `ANSIBLE_SSH_USER=ubuntu`) when you run `./streisand`.
+See also:
+* A [more technical list of features](Features.md) 
+* A [more technical list of services](Services.md)
 
-**Note:** Running Streisand against an existing server can be a destructive action! You will be potentially overwriting configuration files and must be certain that you are affecting the correct machine.
+<a id="cloud-providers"></a>
+## Cloud providers
+* Amazon Web Services (AWS)
+* Microsoft Azure
+* Digital Ocean
+* Google Compute Engine (GCE)
+* Linode
+* Rackspace
 
-### Noninteractive Deployment (Advanced) ###
 
-Alternative scripts and configuration file examples are provided for
-noninteractive deployment, in which all of the required information is passed
-on the command line or in a configuration file.
+#### Other providers
+If you are an expert and can set up a *fresh Ubuntu 16.04 server* elsewhere, Streisand includes code for:
 
-Example configuration files are found under `global_vars/noninteractive`. Copy
-and edit the desired parameters, such as providing API tokens and other choices,
-and then run the appropriate script.
+* Using `ssh` to connect to your remote server from your local machine
+* After connecting to your remote server, you can run Streisand *in place*, called "localhost deployment". This converts the server to a Streisand node.
```

Comment:
I think this bullet and the prev could be condensed into one like:

* Using `ssh` to connect to a remote server to run the Streisand builder *in place*, converting the remote server into a Streisand VPN server. We call this "localhost deployment".

I would avoid the term "Streisand node" since I don't think its used elsewhere in the README to this point.

Reply:
This is split up because of "existing server" vs "localhost" deployment. I've rewritten.

Reply:
OK, I took your advice and just punted everybody to the advanced instructions.